docs/nx/next/react-query.md

```diff
@@ -0,0 +1,81 @@
+<!-- markdownlint-disable MD041 -->
+### @ensono-stacks/next:react-query
+
+<details>
+<summary>Add Next Authentication to your next application</summary>
+
+The react-query generator will install and update `_app.tsx` into an existing Next application. It will update the `_app.tsx` file by adding the query provider and update the `.eslintrc` file with the react-query eslint plugin.
```

Comment:
It will install and update '_app.tsx'? 

This initial description needs to be much more high level, the details you have here should be covered in the generator output.

```suggestion
This generator is used to configure your application with state management, using the plugin (link to plugin repo) react-query. In addition, the application will be updated with the relevant linting rules to ensure that the application conforms is compliant when using react-query.
```